### DISCUSSION TITLE
Installation Prometheus

### DISCUSSION THREAD

**Discussion by msoso38** [sladkoff/minecraft-prometheus-exporter#134] [Q&A] [answered]
where do we install prometheus ??

**Comment by sladkoff** [comment, marked as answer]
You can install Prometheus on your minecraft server host or on any other machine (or inside a Docker / Kubernetes environment).
You just need to make sure that your minecraft server is reachable over the network from your Prometheus instance.

See [Installation](https://prometheus.io/docs/prometheus/latest/installation/) instructions.
